code/ATMOSPHERICS/components/unary_devices/vent_pump.dm

```diff
@@ -3,6 +3,8 @@
 #define PRESSURE_CHECKS 1
 #undefine
 
+var/global/list/all_vent_pumps = list()
```

Comment:
`global` does nothing special in this context, it's to define object static vars.